CHANGELOG.md

```diff
@@ -39,6 +39,8 @@
   For every release, there is now a date, title, and summary.
   This both improves the changelog itself but can then also be used to postprocess the CHANGELOG programmatically.
 
+* `VDSL3`: Add default to the anonymous workflow of a VDSL3 component without adding it to the functionality resources.
```

Comment:
Add default **_id_** the anonymous workflow?

Reply:
Check!